docs/reference/modules/settings.md

```diff
@@ -0,0 +1,127 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/settings`
+
+**Alias:** `apos.settings`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+This module governs the Personal Settings menu that allows users to change their preferences. Apostrophe projects can configure this menu through `@apostrophecms/settings`.
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+| [`subforms`](#subforms) | Object | Each object defines an input field in the Personal Settings menu. |
+| [`groups`](#groups) | Object | Like the property of the same name in the `fields` setting, this option organizes the subforms on tabs in the Personal Settings menu. |
```

Comment:
Use of the word `setting` for anything else in this article is probably too confusing.

Reply:
Changed.